src/main/scala/firrtl/passes/memlib/VerilogMemDelays.scala

```diff
@@ -175,7 +175,7 @@ object VerilogMemDelays extends Pass {
          Dependency[SystemVerilogEmitter] )
 
   override def invalidates(a: Transform): Boolean = a match {
-    case _: transforms.ConstantPropagation | ResolveFlows => true
+    case _: transforms.ConstantPropagation | ResolveFlows | ResolveKinds => true
```

Comment:
This one is interesting because the one `WRef` it seems to create actually appears to have the correct kind (`RegKind`).